test/middleware/session-middleware.test.ts

```diff
@@ -63,7 +63,7 @@ describe("Session middleware", () => {
                 const sessionStore = Substitute.for<SessionStore>();
 
                 await SessionMiddleware(config, sessionStore, createSessionWhenNotFound)(request, Substitute.for<Response>(), nextFunction);
-
+                console.log(request);
```

Comment:
do you still need this ? log the request ?

Reply:
removed!